sla_per_use_system_test.py

```diff
@@ -0,0 +1,208 @@
+                                          result['partition rate'],
+                                          result['row rate'],
+                                          result['latency mean'],
+                                          result['latency median'],
+                                          result['latency 95th percentile'],
+                                          result['latency 99th percentile'],
+                                          result['latency 99.9th percentile'],
+                                          result['Total partitions'],
+                                          result['Total errors']))
+
+    def get_test_xml(self, result, test_name=''):
+        test_content = """
+  <test name="%s: (%s) Loader%s CPU%s Keyspace%s" executed="yes">
+    <description>"%s test, ami_id: %s, scylla version:
+    %s", hardware: %s</description>
+    <targets>
+      <target threaded="yes">target-ami_id-%s</target>
+      <target threaded="yes">target-version-%s</target>
+    </targets>
+    <platform name="AWS platform">
+      <hardware>%s</hardware>
+    </platform>
+
+    <result>
+      <success passed="yes" state="1"/>
+      <performance unit="kbs" mesure="%s" isRelevant="true" />
+      <metrics>
+        <op-rate unit="op/s" mesure="%s" isRelevant="true" />
+        <partition-rate unit="pk/s" mesure="%s" isRelevant="true" />
+        <row-rate unit="row/s" mesure="%s" isRelevant="true" />
+        <latency-mean unit="mean" mesure="%s" isRelevant="true" />
+        <latency-median unit="med" mesure="%s" isRelevant="true" />
+        <l-95th-pct unit=".95" mesure="%s" isRelevant="true" />
+        <l-99th-pct unit=".99" mesure="%s" isRelevant="true" />
+        <l-99.9th-pct unit=".999" mesure="%s" isRelevant="true" />
+        <total_partitions unit="total_partitions" mesure="%s" isRelevant="true" />
+        <total_errors unit="total_errors" mesure="%s" isRelevant="true" />
+      </metrics>
+    </result>
+  </test>
+""" % (test_name, result['loader_idx'],
+            result['loader_idx'],
+            result['cpu_idx'],
+            result['keyspace_idx'],
+            test_name,
+            self.params.get('ami_id_db_scylla'),
+            self.params.get('ami_id_db_scylla_desc'),
+            self.params.get('instance_type_db'),
+            self.params.get('ami_id_db_scylla'),
+            self.params.get('ami_id_db_scylla_desc'),
+            self.params.get('instance_type_db'),
+            result['op rate'],
+            result['op rate'],
+            result['partition rate'],
+            result['row rate'],
+            result['latency mean'],
+            result['latency median'],
+            result['latency 95th percentile'],
+            result['latency 99th percentile'],
+            result['latency 99.9th percentile'],
+            result['Total partitions'],
+            result['Total errors'])
+
+        return test_content
+
+    def display_results(self, results, test_name=''):
```

Comment:
looks like it is a copy-paste from `performance_regression_test.PerformanceRegressionTest`